### PR TITLE
VAL-530: Check Aragon ACL alerts for Simple DVT (Forta)

### PR DESCRIPTION
**Pull request by AlexandrMov**
- simpledvt alerting
- holesky testnet support

### CODE REVIEW

**Review by dgusakov — CHANGES_REQUESTED** [review]

File: ethereum-governance/e2e/__snapshots__/agent-proxy-watcher.e2e-spec.ts.snap [hunk 1]
```diff
@@ -19,7 +19,7 @@ exports[`agent-proxy-watcher e2e tests should process block with changed proxy i
   Finding {
     "addresses": [],
     "alertId": "PROXY-IMPL-CHANGED",
-    "description": "Implementation of Legacy Oracle ([0x442af784a788a5bd6f42a01ebe9f287a871243fb](https://etherscan.io/address/0x442af784a788a5bd6f42a01ebe9f287a871243fb)) changed from [0xADa6E33aa65590934870bFD204Fe8efe9c6Aa4bC](https://etherscan.io/address/0xADa6E33aa65590934870bFD204Fe8efe9c6Aa4bC) to [0x1430194905301504e8830ce4B0b0df7187E84AbD](https://etherscan.io/address/0x1430194905301504e8830ce4B0b0df7187E84AbD)",
+    "description": "Implementation of Curated NO Registry ([0x55032650b14df07b85bf18a3a3ec8e0af2e028d5](https://etherscan.io/address/0x55032650b14df07b85bf18a3a3ec8e0af2e028d5)) changed from [0xec3567ae258639a0FF5A02F7eAF4E4aE4416C5fe](https://etherscan.io/address/0xec3567ae258639a0FF5A02F7eAF4E4aE4416C5fe) to [0x5d39ABaa161e622B99D45616afC8B837E9F19a25](https://etherscan.io/address/0x5d39ABaa161e622B99D45616afC8B837E9F19a25)",
```
[the remark this answers is not in the document]
Comment (AlexandrMov) [reply ethereum-governance/e2e/__snapshots__/agent-proxy-watcher.e2e-spec.ts.snap]: The test already has ordering by description. I can suggest whether to adjust the name of alert only or change snapshot directly.

File: ethereum-governance/src/common/constants.testnet.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ export const LIDO_LOCATOR_ADDRESS =
   "0x1eDf09b5023DC86737b59dE68a8130De878984f5";
 export const LIDO_STETH_ADDRESS = "0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F";
 export const EASY_TRACK_ADDRESS = "0xAf072C8D368E4DD4A9d4fF6A76693887d6ae92Af";
-export const NODE_OPERATORS_REGISTRY_ADDRESS =
+export const CURATED_NODE_OPERATORS_REGISTRY_ADDRESS =
```
[the remark this answers is not in the document]
Comment (AlexandrMov) [reply ethereum-governance/src/common/constants.testnet.ts:10]: I renamed those files

**Review by dgusakov — APPROVED** [review]

**Comment by sergeyWh1te** [comment]
My main question for all pull requests. Are there unit tests?

**Comment by AlexandrMov** [comment]
> My main question for all pull requests. Are there unit tests?

There no e2e tests due to simpleDVT has not been deployed on Mainnet yet. I thought to add e2e tests once the new module is deployed. I suggest it would be a separate task.

**Review by infloop — APPROVED** [review]

**Review by BATMAH69 — APPROVED** [review]

File: ethereum-governance/src/subagents/easy-track/constants.holesky.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+export const SIMPLEDVT_NODE_OPERATORS_REGISTRY_ADDRESS = simpleDvtNorAddress;
+
+export const EASY_TRACK_EVENTS_OF_NOTICE = mainnetEventsOfNotice.map(
+  (event) => ({
+    ...event,
+    address: EASY_TRACK_ADDRESS,
+  }),
+);
+
+export const EASY_TRACK_TYPES_BY_FACTORIES = new Map<string, string>([
```
Comment (BATMAH69) [inline ethereum-governance/src/subagents/easy-track/constants.holesky.ts:25]: looks like addresses from goerly

**Review by BATMAH69 — CHANGES_REQUESTED** [review]

File: ethereum-governance/src/subagents/easy-track/constants.holesky.ts (hunk 3 above)
Comment (AlexandrMov) [reply ethereum-governance/src/subagents/easy-track/constants.holesky.ts:25]: We had a conversation about easy tracks. That the gov bot owners will upgrade easy tracks themselves. I just added the holesky network supporting.

**Review by BATMAH69 — APPROVED** [review]

**Pull request merged**